.github/workflows/pull-request.yaml

```diff
@@ -287,25 +282,20 @@ jobs:
         run: |
           kubectl port-forward \
             --namespace tracetest-integration \
-            svc/tracetest-integration 8080 &
+            svc/tracetest-integration 11633:8080 &
```

Comment:
Won't this cause a problem with the CLI? You are configuring the CLI to send commands to `localhost:11633`, but here tracetest will be available on port `8080`.